apps/marketplace/actions/briefActions.js

```diff
@@ -317,6 +347,83 @@ export const handleBriefResponseSubmit = (briefId, model) => (dispatch, getState
   })
 }
 
+export const handleCreateBriefResponseSuccess = response => ({
+  type: BRIEF_RESPONSE_CREATE_SUCCESS,
+  createdBriefResponse: response.data
+})
+
+export const createBriefResponse = briefId => (dispatch, getState) => {
+  dispatch(sendingRequest(true))
+  return dmapi({
+    url: `/brief/${briefId}/respond`,
+    method: 'POST',
+    headers: {
+      'X-CSRFToken': getState().app.csrfToken,
+      'Content-Type': 'application/json'
+    }
+  }).then(response => {
+    if (response.error) {
+      dispatch(handleErrorFailure(response))
+    } else {
+      dispatch(handleCreateBriefResponseSuccess(response))
+    }
+    dispatch(sendingRequest(false))
+    return response
+  })
+}
+
+export const handleSaveBriefResponse = () => ({
+  type: BRIEF_RESPONSE_SAVE
+})
+
+export const resetBriefResponseSave = () => ({
+  type: BRIEF_RESPONSE_SAVE_RESET
+})
+
+export const saveBriefResponse = (briefId, briefResponseId, model) => (dispatch, getState) => {
+  dispatch(sendingRequest(true))
+  dmapi({
+    url: `/brief/${briefId}/respond/${briefResponseId}`,
+    method: 'PATCH',
+    headers: {
+      'X-CSRFToken': getState().app.csrfToken,
+      'Content-Type': 'application/json'
+    },
+    data: JSON.stringify(model)
+  }).then(response => {
+    if (response.error) {
+      dispatch(handleErrorFailure(response))
+    } else {
+      dispatch(handleBriefResponseSuccess(response))
```

Comment:
This is correct as is - the `handleBriefResponseSuccess` is called because the response contains the updated brief response from the server, and the `handleSaveBriefResponse` dispatch is happening in `BriefResponsePage` when the response is saved but not submitted.